src/test_workflow/benchmark_test/benchmark_test_suite.py

```diff
@@ -65,7 +70,9 @@ def __init__(
                 self.command += f" --telemetry-params '{self.args.telemetry_params}'"
 
     def execute(self) -> None:
-        if self.security:
+        if self.security and semver.compare(self.distribution_version, '2.12.0') != -1:
+            self.command += ' --client-options="timeout:300,use_ssl:true,verify_certs:false,basic_auth_user:\'admin\',basic_auth_password:\'myStrongPassword123!\'"'
+        elif self.security:
```

Comment:
Why not pass the password as per version instead of repeating the command?
```suggestion
        if self.security:
           password = admin
            if semver.compare(self.distribution_version, '2.12.0') != -1:
               password = myStrongPassword123
           self.command += ' --client-options="timeout:300,use_ssl:true,verify_certs:false,basic_auth_user:\'admin\',basic_auth_password:\'${password}!\'"'
```

Reply:
makes sense. Updated.